README.md

```diff
@@ -37,6 +37,18 @@ The following software requirements are necessary to run the Bold Reports Enterp
 * Application: Docker
 * Web Browser: Microsoft Edge, Mozilla Firefox, and Chrome
 
+# Supported tags
+
+| Tags               | OS Version    | Last Modified |
+| -------------      | ------------- | ------------- |
+| `4.2.78`           | Ubuntu 20.04  (amd64)    | 07/02/2023 |
+| `4.2.78_alpine`    | Alpine 3.13  (amd64)  | 07/02/2023 |
```

Comment:
is these tags where available in GCR?

Reply:
@Velmurugan-Thangaraj , Now I have removed the not avaible tags in gcr